graylog2-server/src/main/java/org/graylog2/security/ldap/LdapConnector.java

```diff
@@ -326,7 +326,7 @@ private String normalizedDn(String dn) {
             try {
                 return new Dn(dn).getNormName();
             } catch (LdapInvalidDnException e) {
-                LOG.warn("Invalid DN", e);
+                LOG.debug("Invalid DN", e);
```

Comment:
Fair enough. Could you please add a small Javadoc comment to the `normalizeDn()` method which says exactly this?

This way future me at least won't scratch his head about it. 😉